src/datascience-ui/native-editor/nativeEditor.tsx

```diff
@@ -98,12 +99,18 @@ export class NativeEditor extends React.Component<INativeEditorProps> {
                 />
             );
 
+        // Use markdown to render the banner
+        const BannerTransform = getTransform('text/markdown');
+
         return (
             <div id="main-panel" role="Main" style={dynamicFont}>
                 <div className="styleSetter">
                     <style>{`${this.props.rootCss ? this.props.rootCss : ''}
 ${buildSettingsCss(this.props.settings)}`}</style>
                 </div>
+                <div className="custom-editor-banner">
+                    <BannerTransform data={getLocString('DataScience.customEditorBanner', 'This is going away')}/>
```

Comment:
Does this need to match the package.nls.json entry?

Reply:
Only if there's a bug. Package.nls.json should always work.